packages/esm-patient-registration-app/src/patient-registration/patient-registration-utils.ts

```diff
@@ -103,7 +103,7 @@ export function scrollIntoView(viewId: string) {
 }
 
 export function cancelRegistration() {
-  navigate({ to: `${window.spaBase}/home` });
+  window.history.back();
```

Comment:
Defer to @ibacher and @denniskigen as to whether this fits in with normal standards.  Will this result in a page reload?

Reply:
This doesn't do a full-page reload. It is shown best when registering a patient and routing back to `/home`. For the patient chart, it isn't necessarily a page reload since the patient, orders, visits endpoints are hit every time a user is routed to the patient chart

Reply:
Works for me @Jexsie , but would defer to another reviewer as to whether this is the right approach.  I'll add @ibacher as a reviewer.